src/scss/init/_index.scss

```diff
@@ -28,7 +28,7 @@
 // - [Breakpoint](http://breakpoint-sass.com/)
 // - [Chroma-sass](http://johnalbin.github.io/chroma/)
 // - [Compass vertical rhythm](http://compass-style.org/)
-// For compass we we only load vertical rhythm support
+// For compass we we only load vertical rhythm support.
```

Comment:
Duplicated word 'we', can you amend to just 'For compass we only load vertical rhythm support.'